src/com/lundekhan/stokk/StokkHelper.kt

```diff
@@ -0,0 +1,24 @@
+package com.lundekhan.stokk
+
+import kotlin.math.pow
+
+object StokkHelper {
+    fun compoundInterest(years: Int, startAmount: Double, interest: Double, monthlyDeposit: Int): Double =
+        (0 until years*12).fold(startAmount) { acc, _ -> (acc + monthlyDeposit) * interest.pow(1.0/12) }
```

Comment:
1.0/12 could be extracted into a variable name (even though compiler solves this, it looks better)